### DISCUSSION TITLE
How about adding some color to Copilot?

### DISCUSSION THREAD

**Discussion by CoralPink** [rmehri01/onenord.nvim#69] [Ideas]
Hello!

Thanks for the last time!
This time I found a rather interesting topic. 😉

This screenshot is from using `nvim-cmp` + `copilot.nvim`.

<img width="863" alt="Copilot" src="https://github.com/rmehri01/onenord.nvim/assets/30016220/1fe726f8-6da3-4349-9ef6-0cb504eba383">

plus `lspkind.nvim`

<img width="581" alt="copilot-icon" src="https://github.com/rmehri01/onenord.nvim/assets/30016220/8c13667f-1852-4598-997a-38a91c3409de">

Ah! It is not colored against `Copilot`! 🙀

I have no problem with it as it is, but I thought it might be interesting to consider a different color.

What do you think?

refs:
https://github.com/hrsh7th/nvim-cmp
https://github.com/zbirenbaum/copilot.lua
https://github.com/onsails/lspkind.nvim

**Comment by rmehri01** [comment]
Hey again! I just pushed 3b3ae8a1f32d9f735b5c9ccf287b73e0f5e6d7a5 which should fix this issue, at least it works for me with `copilot-cmp`, which isn't the exact same setup but I think should be similar. If you have any issues let me know!

**Comment by CoralPink** [comment]
Yup, it's clear!
(I'm a little unsure, but is this the right English expression? 😅)

<img width="226" alt="copilot-icon" src="https://github.com/rmehri01/onenord.nvim/assets/30016220/a0e3016e-7243-42f1-bc19-1d03c5c0c8c9">

Thank you!!

I will continue to support you! 😆

**Reply by rmehri01** [reply]
Not sure if that is the right English expression either, maybe "it's resolved" or "it's fixed" but I think I get the point 😄

No problem, thank you for your support!

**Comment by CoralPink** [comment]
Oh, I knew it…😮

Anyway, thank you very much❗
